src/modules/governance/FormattedProposalTime.tsx

```diff
@@ -93,7 +115,7 @@ export function FormattedProposalTime({
         {canBeExecuted ? <Trans>Expires</Trans> : <Trans>Can be executed</Trans>}
         &nbsp;
       </Typography>
-      {dayjs.unix(canBeExecuted ? executionTimeWithGracePeriod : executionTime).fromNow()}
+      {dayjs.unix(canBeExecuted ? executionTimeWithGracePeriod : exectutionTimeStamp).fromNow()}
```

Comment:
The `executionTimeWithGracePeriod` should also account for the cross-chain delay. `executionTime` is the start of the window to execute, `executionTimeWithGracePeriod` is the the time where proposal expires and can no longer be executed.

Reply:
Discussed earlier that it makes sense to use a new state, as the delay occurs after execution, not between queued and exeuction